boards/others/esp32c3_supermini/doc/index.rst

```diff
@@ -0,0 +1,281 @@
+.. zephyr:board:: esp32c3_supermini
+
+ESP32-C3-SUPERMINI
+##################
```

Comment:
Apologies if I missed this earlier or if I made an incorrect request for change at some point but this needs to go as it's redundant with what the zephyr:board directive automatically adds already


```suggestion
```

Reply:
Will do.